Monika-After-Story/MonikaModDev
# Clean Hairs + orca hair spritepacks

#3972 #3786 

wait for #4582 before testing

# Key changes
* cleaned up hair from @orcaramelo
* bun hair spritepack
* 4 new hair styles (spritepack) from @orcaramelo
    1. ponytail with braid
    2. twin bun
    3. twin tails
    4. usagi

# Followups
* this needs thumbnails, @jmwall24 

# Testing
* verify existing hair looks good
* verify bun hair spritepack loads and is selectable in hairstyle selector
* verify orcaramelo hair spritepack loads and is selectable in hairstyle selector
